firestore-bigquery-export/extension.yaml

```diff
@@ -338,3 +338,7 @@ params:
     example: https://us-west1-my-project-id.cloudfunctions.net/myTransformFunction
     type: string
     required: false
+
+events:
+  - type: com.bigqueryexport.v1.sync.start
```

Comment:
Should this not be `firebase.extensions.big-query-export`?

Reply:
Yes, this will need updating. We have an example `firebase.extensions.storage-resize-images.v1.complete`